src/System.Private.CoreLib/shared/System/Threading/AsyncLocal.cs

```diff
@@ -61,7 +62,7 @@ public T Value
             get
             {
                 object obj = ExecutionContext.GetLocalValue(this);
-                return (obj == null) ? default : (T)obj;
+                return (obj == null) ? default! : (T)obj; // TODO-NULLABLE: Need a way to annotate that Value may return null
```

Comment:
Should we probably add like a generic reference to these comments? So once the generic stuff lands we can grep on all this comments and fix those? Something like `// TODO-NULLABLE-GENERIC`?

Reply:
Done